slackbot/manager.py

```diff
@@ -5,15 +5,14 @@
 from glob import glob
 from six import PY2
 from importlib import import_module
-from slackbot import settings
 from slackbot.utils import to_utf8
 
 logger = logging.getLogger(__name__)
 
 
 class PluginsManager(object):
-    def __init__(self):
-        pass
+    def __init__(self, settings):
+        self.settings = settings
```

Comment:
To allow for multiple bots the Plugin Manager needed to be rewritten to read the plugins from the passed settings file, instead of grabbing a static one declared in this python script